packages/core/src/scp-cf/env-destination-accessor.ts

```diff
@@ -1,22 +1,31 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
+import { createLogger } from '@sap-cloud-sdk/util';
 import { addProxyConfigurationInternet, ProxyStrategy, proxyStrategy } from '../util/proxy-util';
-import { sanitizeDestination } from './destination';
+import { sanitizeDestination, isDestinationConfiguration, parseDestination } from './destination';
 import { Destination } from './destination-service-types';
 
+const logger = createLogger({
+  package: 'core',
+  messageContext: 'env-destination-accessor'
+});
+
 /**
  * Get all destinations from the environment variable "destinations".
- * This is discouraged for productive use! Use destination-accessor/useOrFetchDestination for fetching destinations
- * from the Cloud Foundry destination service.
+ * This is discouraged for productive use! Use [[useOrFetchDestination]] for fetching destinations from the Cloud Foundry destination service.
  *
  * @returns A list of destinations
  */
 export function getDestinationsFromEnv(): Destination[] {
-  if (getDestinationsEnvVariable()) {
-    return (JSON.parse(getDestinationsEnvVariable()!) as any[]).map(entry => sanitizeDestination(entry));
-  } else {
-    return [];
+  const destinationsEnv = getDestinationsEnvVariable();
```

Comment:
I know it is potentially dangerous to log destination information so we do not want encourage customers to do that. But in order to investigate failing cases perhaps it makes sense to add a trace or info level log statement logging the content of the enviorment variable for investigation purpose.